internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -594,8 +596,21 @@ func (rmq *rocksmq) Produce(topicName string, messages []ProducerMessage) ([]Uni
 		msgID := idStart + UniqueID(i)
 		key := path.Join(topicName, strconv.FormatInt(msgID, 10))
 		batch.Put([]byte(key), messages[i].Payload)
+		var pLen int
+		properties, err := json.Marshal(messages[i].Properties)
+		if err != nil {
+			log.Warn("properties marshal failed",
+				zap.Int64("msgID", msgID),
+				zap.String("topicName", topicName))
```

Comment:
add `zap. Error(err)`, and should return error here, because `Consume()` requires `properties` must exist